listener/syncer.go

```diff
@@ -264,6 +266,7 @@ func (s *Syncer) processEvents(logs []ethTypes.Log, header ethTypes.Header) {
 				case "DepositsFinalised":
 					s.processDepositFinalised(selectedEvent.Name, &abiObject, &vLog)
 				default:
+					fmt.Println("Default here")
```

Comment:
What's the difference between using `fmt.Println` and `s.Logger.Debug`?

Reply:
Just that it `s.Logger.Debug` prints the log in the standard of the application whereas `fmt.Println` is just dumped on stdOutput. I intend to remove all `fmt` statements in the next PR's